pkg/rancher-desktop/backend/lima.ts

```diff
@@ -607,16 +607,22 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     const mounts: LimaMount[] = [];
     const locations = ['~', '/tmp/rancher-desktop'];
```

Comment:
If we put the home directory in here, then we can simplify the rest of the logic because we no longer need `homeDir`:

```suggestion
    const locations = [process.env.HOME, '/tmp/rancher-desktop'];
```